src/color.rs

```diff
@@ -30,38 +30,28 @@ pub enum ColorType {
     /// Pixel is 8-bit BGR with an alpha channel
     Bgra8,
 
-    /// Pixel is of unknown color type with the specified bits per pixel. This can apply to pixels
-    /// which are associated with an external palette. In that case, the pixel value is an index
-    /// into the palette.
-    Unknown(u8),
-
     #[doc(hidden)]
     __Nonexhaustive,
 }
 
-/// Returns the number of bits contained in a pixel of `ColorType` ```c```
-pub fn bits_per_pixel(c: ColorType) -> usize {
-    match c {
-        ColorType::L8 => 8,
-        ColorType::L16 => 16,
-        ColorType::La8 => 16,
-        ColorType::Rgb8 | ColorType::Bgr8 => 24,
-        ColorType::Rgba8 | ColorType::Bgra8 | ColorType::La16 => 32,
-        ColorType::Rgb16 => 48,
-        ColorType::Rgba16 => 64,
-        ColorType::Unknown(n) => n as usize,
-        ColorType::__Nonexhaustive => unreachable!(),
+impl ColorType {
+    /// Returns the number of bits contained in a pixel of `ColorType` ```c```
+    pub fn bytes_per_pixel(self) -> usize {
```

Comment:
Make it `u8`, as we agreed would be the a useful range.

Reply:
Also, run clippy after that change because I'm sure it will turn up many instances of conversion that could be turned into `_::from` if the type is not platform dependent.